fatal error on npm install

```
npm ERR! Darwin 14.3.0
npm ERR! argv "/Users/ag_dubs/.nvm/versions/node/v0.12.6/bin/node" "/Users/ag_dubs/.nvm/versions/node/v0.12.6/bin/npm" "install"
npm ERR! node v0.12.6
npm ERR! npm  v2.11.2
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package phantomjs does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer mocha-phantomjs@3.6.0 wants phantomjs@1.9.1 - 1.9.7-15

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ag_dubs/Projects/id.webmaker.org/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -93,7 +93,7 @@
     "mofo-style": "latest",
     "ncp": "^2.0.0",
     "npm-run-all": "^1.1.3",
-    "phantomjs": "^1.9.16"
+    "phantomjs": "1.9.7-15"
   },
   "engines": {
     "npm": "^2.5.1",

